falcon/errors.py

```diff
@@ -952,8 +952,8 @@ def __init__(
         )
 
 
-class HTTPPayloadTooLarge(HTTPError):
-    """413 Payload Too Large.
+class HTTPContentTooLarge(HTTPError):
```

Comment:
We must keep the old name for compatibility. Even though 4.0 is a major release that affords breaking changes, we must allow for a longer deprecation window for the publicly documented stuff.